zebrad/tests/acceptance.rs

```diff
@@ -544,15 +544,18 @@ fn version_args() -> Result<()> {
     let testdir = testdir()?.with_config(&mut default_test_config()?)?;
     let testdir = &testdir;
 
-    // unexpected free argument `argument`
-    let child = testdir.spawn_child(args!["version", "argument"])?;
-    let output = child.wait_with_output()?;
-    output.assert_failure()?;
-
     // unrecognized option `-f`
```

Comment:
This comment is out of date, it doesn't match the outcome of the test any more. Could tip-height be used here instead?

Allowing unrecognised arguments after version is potentially a breaking change, but it's minor.

Reply:
> This comment is out of date, it doesn't match the outcome of the test any more. Could tip-height be used here instead?

I used `tip-height` instead and added "is ignored" to the comment for this check in https://github.com/ZcashFoundation/zebra/commit/6b553d290759d64255389c43c1dfb6698313b28b.

> Allowing unrecognised arguments after version is potentially a breaking change, but it's minor.

I'll leave that in the PR description.